samples/Exporters/Console/TestOtlp.cs

```diff
@@ -47,5 +57,28 @@ internal static object Run(string endpoint)
 
             return null;
         }
+
+        private static object RunWithActivitySource(string endpoint)
+        {
+            // Enable OpenTelemetry for the source "MyCompany.MyProduct.MyWebServer"
+            // and use Console exporter
+            OpenTelemetrySdk.EnableOpenTelemetry(
+                builder => builder
+                    .AddActivitySource("Samples.SampleServer")
```

Comment:
@cijothomas I'm leaning into listening to all sources by default and filter out un-wanted ones. Depending on how some libraries are instrumented we may need some processors to filter out some activities.

Reply:
Yup. Lets address this separately.
Libraries like httpclient, which was instrumented using DiagnosticSource, will have their activities emitted with empty ActivitySource. Listening to them requires `AddActivitySource("")` now. 
Anyway, lets address this after some exporters/adapters are ready with Activity API.